pangeo-notebook/packages.txt

```diff
@@ -220,7 +220,7 @@ jupyter_server-1.10.1
 jupyter_telemetry-0.1.0
 jupyterhub-base-1.4.2
 jupyterhub-singleuser-1.4.2
-jupyterlab-3.0.16
+jupyterlab-3.1.0
```

Comment:
Check that packages are bumped up, and key ones like xarray, jupyterlab, dask are not downgraded in the packages.txt file